### ISSUE TITLE
Migrate functionality from run_smrtlink_job.py to pbservice

### ISSUE THREAD

**Issue by mpkocher** [PacificBiosciences/smrtflow#129]
There's an internal tool `run_smrtlink_job.py`, that runs pipelines from files, not analysis.json files.

It's probably useful to fold the functionality back into `pbservice`.

Something similar to how pbsmrtpipe works.

`pbservice run-analysis {pipeline-template-full-id-or-short-name} -e:eid_subreadset:"/path/to/subreadset" -e:eid_ref:"/path/to/refereneceset.xml" --preset.xml --host=smrtlink-beta --port=8081 --block`

Where the pipeline template id is assumed to be complete form (`pbsmrtpipe.pipelines.my_pipeline`), or a short form, `my_pipeline`

The current subparser `run-analysis` might be better renamed to `run-analysis-json`?

**Comment by mpkocher** [comment]
@natechols can you comment on this?

**Comment by natechols** [comment]
Very easy to implement, I can add it tomorrow, but the use of tags like `eid_subread` is one of the things that the Python tool was meant to avoid - even I can't remember what these are - so I'd prefer to just use the syntax `-e /path/to/subreadset.xml` and have the entry point ID inferred automatically.

**Comment by mpkocher** [comment]
Sounds good.

So the new comandline interface with be --subreadset (-s) --referenceset (-r ) --barcodeset (-b) for the entry points? 

```
pbservice run-analysis {pipeline-template-full-id-or-short-name} \
-s /path/to/subreadset .xml \
-r /path/to/refereneceset.xml \
 --preset.xml  \
--host=smrtlink-beta --port=8081 --block
```

For non Resequencing based pipelines, the analysis.json should be used.

**Comment by natechols** [comment]
Well, after sleeping on it, I'm not sure it's so great to diverge from the pbsmrtpipe argument syntax; for now it's probably safer to mimic it as closely as possible.  My complaint about `eid_*` applies equally to pbsmrtpipe too - I think handling this implicitly would make both programs easier to use.  Obviously it won't cover pipelines where the entry points have multiple datasets of the same type, but I don't think we've implemented any yet, and of course that's where the JSON comes in handy.

Anyway, I'll go ahead and add the essential functionality and we can revisit parameter syntax later.  BTW the pipeline names were the other reason for the Python script - I can never remember these either.  (But I'll add some help text to point users to the appropriate pbsmrtpipe command for showing all pipelines.)

**Comment by natechols** [comment]
Fixed in 695ca499ed87dba42b60da4141c56fc0c9d617a2
